backend/compat.py

```diff
@@ -1 +1,13 @@
+# Werk van Sil
 
+# for-loop
+# [] 
+
+
+def validate_appointment(data: dict):
+    KeyList = ['title', 'description', ...]
+    for key in KeyList:
```

Comment:
De `for` loop ging wat stroef. Het is waarschijnlijk handig als je eens kijkt op codecedemy.

Daarnaast zou ik graag willen dat je de volgende vragen maakt:
- De algemene syntax van de `for`-loop is `for x in y`. Wat is `x`? En wat stelt `y` voor?
- Wat doet `range`?
- Wat doet `enumerate`?
- Wat doet `for key in {1: 2, 3: 4}: print(key)`?
- Hoe zou je de values in de dictionary van de vorige vraag printen?
- Wat doet `for item in [1, 2, 3, 4]: print(item)`?

Je kunt de vragen wel maken in een apart bestand en dat uploaden (help ik wel even mee).